src/main/java/uk/gov/hmcts/reform/migration/ccd/CoreCaseDataService.java

```diff
@@ -119,44 +110,12 @@ CaseDetails apply(
             return null;
         }
 
-        final Optional<Dtspb4583Dates> caseDatesOpt = dtspb4583DataService.get(caseDetails.getId());
-        if (caseDatesOpt.isEmpty()) {
-            log.warn("Case {}: Attempting to update without entry in DTSPB-4583 data", caseDetails.getId());
-            return null;
-        }
-        final Dtspb4583Dates caseDates = caseDatesOpt.get();
-
         final Map<String, Object> updatedData = caseDetails.getData();
 
-        final Object currentSubmDateObj = updatedData.get(APPL_SUBMIT_DATE);
-        if (currentSubmDateObj == null) {
-            log.warn("Case {}: No application submitted date found in case data", caseDetails.getId());
-            return null;
+        if (!isRollback) {
+            updatedData.put(AUTO_CLOSED_EXPIRY, YES);
```

Comment:
need to check the autoClosedExpiry!=Yes from caseDetails from startEventResponse before update to Yes